tests/test_integrity.py

```diff
@@ -476,6 +476,239 @@ def test_sources(db):
+        {'source': '2000-7523', 'value': '1.88 +or- 0.11', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2047-0718', 'value': '0.96 +or- 0.17', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2057-0252', 'value': '1.02 +or- 0.07', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2101+1756', 'value': 'None', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2104-1037', 'value': '1.02 +or- 0.07', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2114-2251', 'value': '1.41 +or- 0.03', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2126-8140', 'value': '1.39 +or- 0.19', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2127-4215', 'value': '1.21 +or- 0.13', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2139+0220', 'value': '0.96 +or- 0.16', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2144+1446', 'value': '1.17 +or- 0.05', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2148+4003', 'value': '0.99 +or- 0.10', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2206-4217', 'value': '1.33 +or- 0.10', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2208+2921', 'value': '1.41 +or- 0.20', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2224-0158', 'value': '0.99 +or- 0.08', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2228-4310', 'value': '0.94 +or- 0.16', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2234+4041', 'value': 'None', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2237+3922', 'value': '1.04 +or- 0.06', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2244+2043', 'value': '1.29 +or- 0.03', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2306-0502', 'value': '1.14 +or- 0.04', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2322-3133', 'value': '1.39 +or- 0.19', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2322-6151', 'value': '1.59 +or- 0.05', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2351-2537', 'value': '1.16 +or- 0.11', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ,
+        {'source': '2354-3316', 'value': '1.09 +or- 0.05', 'parameter': 'radius', 'publication': 'Fili15', 'unit': '(R_Jup)'} ])  # change to csv
```

Comment:
This is the beginning of the ingest. Just keep data for one source and put the rest into a CSV file and save for later.

Let's also add the other parameters for the same source.